go/systests/stellar_client_test.go

```diff
@@ -469,3 +469,26 @@ func (s *stellarRetryClient) MarkAsReadLocal(ctx context.Context, arg stellar1.M
 	}
 	return err
 }
+
+func (s *stellarRetryClient) IsAccountMobileOnlyLocal(ctx context.Context, arg stellar1.IsAccountMobileOnlyLocalArg) (bool, error) {
+	var err error
+	var mobileOnly bool
+	for i := 0; i < retryCount; i++ {
+		mobileOnly, err = s.cli.IsAccountMobileOnlyLocal(ctx, arg)
+		if err == nil {
+			break
+		}
+	}
+	return mobileOnly, err
+}
+
+func (s *stellarRetryClient) SetAccountMobileOnlyLocal(ctx context.Context, arg stellar1.SetAccountMobileOnlyLocalArg) error {
+	var err error
+	for i := 0; i < retryCount; i++ {
+		err = s.cli.SetAccountMobileOnlyLocal(ctx, arg)
+		if err == nil {
+			break
+		}
+	}
+	return err
+}
```

Comment:
I've never seen stellarRetryClient before but why retry rpcs that hit app and stellard but not horizon?

Reply:
well, clients don't know what is going to hit horizon...